digitalpublicgoods/kotani-pay.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "Kotani Pay",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/Kotani-Pay/Kotani-Pay-API/blob/main/LICENSE.txt"
```

Comment:
This `LICENSE.txt` link doesn't exist anymore

Reply:
Flagged in Kotani-Pay/Kotani-Pay-API#27.

Reply:
@jwflory  re -adding it

Reply:
Resolved in Kotani-Pay/Kotani-Pay-API#28.